pkg/karmadactl/cmdinit/karmada/webhook_configuration.go

```diff
@@ -214,6 +214,34 @@ webhooks:
     sideEffects: None
     admissionReviewVersions: ["v1"]
     timeoutSeconds: 3
+  - name: federatedhpa.karmada.io
+    rules:
+      - operations: ["CREATE", "UPDATE"]
+        apiGroups: ["autoscaling.karmada.io"]
+        apiVersions: ["*"]
+        resources: ["federatedhpas"]
+        scope: "Namespaced"
+    clientConfig:
+      url: https://karmada-webhook.%[1]s.svc:443/validate-federatedhpa
+      caBundle: %[2]s
+    failurePolicy: Fail
+    sideEffects: None
+    admissionReviewVersions: [ "v1" ]
+    timeoutSeconds: 10
```

Comment:
The timeout configuration that is too long may tolerate some unreasonable configurations being accepted, and I think 3 seconds is long enough. Can we be consistent with the other timeout configurations at 3 seconds?

Reply:
+1